pkg/plugin/healthz.go

```diff
@@ -94,14 +141,33 @@ func (h *HealthZ) ServeHTTP(w http.ResponseWriter, _ *http.Request) {
 
 // checkRPC initiates a grpc request to validate the socket is responding
 // sends a KMS VersionRequest and checks if the VersionResponse is valid.
-func (h *HealthZ) checkRPC(ctx context.Context, client pb.KeyManagementServiceClient) error {
-	v, err := client.Version(ctx, &pb.VersionRequest{})
+func (h *HealthZ) checkRPC(
+	ctx context.Context,
+	client kmsv1.KeyManagementServiceClient,
+	kmsV2Client kmsv2.KeyManagementServiceClient,
+) error {
+	// v1 check
```

Comment:
I purposely add these comments so that it's easy to read and interpret the code rather than having to figure it out based on var names (in this case) etc.